index.js

```diff
@@ -115,7 +115,13 @@ class SideMenu extends Component {
    */
   handlePanResponderMove(e: Object, gestureState: Object) {
     if (this.state.left.__getValue() * this.menuPositionMultiplier() >= 0) {
-      this.state.left.setValue(this.prevLeft + gestureState.dx);
+      let newLeft = this.prevLeft + gestureState.dx;
+
+      if (!this.props.bounceBackOnOverdraw && Math.abs(newLeft) > this.props.openMenuOffset) {
```

Comment:
PS. You can also set it to false by default, does not matter as long as it can be changed.